gpytorch/settings.py

```diff
@@ -420,6 +420,21 @@ class max_cg_iterations(_value_context):
     _global_value = 1000
 
 
+class min_fixed_noise(_dtype_value_context):
+    """
+    The minimum noise value that can be used in :obj:`~gpytorch.likelihoods.FixedNoiseGaussianLikelihood`.
+    If the supplied noise values are smaller than this, they are rounded up and a warning is raised.
+
+    - Default for `float`: 1e-6
+    - Default for `double`: 1e-10
+    - Default for `half`: 1e-3
```

Comment:
this doesn't seem to agree with the values below?

Reply:
also 1e-8 is pretty small... seems like this wouldn't resolve the issues we've seen with numerical issues creeping up in the backward pass?

Reply:
1e-8 seemed to work for our particular case, but swapped to 1e-6 / 1e-4 for more conservative values.